Library/Formula/azure-cli.rb

```diff
@@ -0,0 +1,36 @@
+class AzureCli < Formula
+  desc "Official Azure CLI"
+  homepage "https://github.com/azure/azure-xplat-cli"
+  url "https://github.com/azure/azure-xplat-cli/releases/download/v0.9.13-December2015/azure-cli.0.9.13.tar.gz"
+  sha256 "f91e341c499624d6ba007ef3ad79d932c5b94a1f35acc49e3fbffd76c2317317"
+
+  head "https://github.com/azure/azure-xplat-cli.git", :branch => "dev"
+
+  option "without-completion", "Azure CLI bash completion will not be installed"
+
+  depends_on "node"
+
+  def install
+    ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+    # install node dependencies
+    `npm install`
```

Comment:
Yeah... omelette is used to generate the cli completion. This will likely change in the not so distant future.

Anyway, the system stuff is dealt with.

Thank you for the feedback, @DomT4, @bfontaine and @mikemcquaid.